src/challenges.js

```diff
@@ -1,49 +1,123 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(param1, param2) {
+  return param1 && param2;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  return (base * height) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(sentence) {
+  return sentence.split(' ');
 }
 
 // Desafio 4
-function concatName() {
-  // seu código aqui
+function concatName(names) {
+  return `${names[names.length - 1]}, ${names[0]}`;
 }
 
 // Desafio 5
-function footballPoints() {
-  // seu código aqui
+function footballPoints(wins, ties) {
+  return (wins * 3) + ties;
 }
 
 // Desafio 6
-function highestCount() {
-  // seu código aqui
+function orderNumbers(numberArray) {
+  return numberArray.sort(function (a, b) { return a - b; });
+}
+
+function highestCount(numberArray) {
+  let array = orderNumbers(numberArray);
+  let counter = 0;
+  for (let index = 0; index < array.length; index += 1) {
```

Comment:
O laço for/in faz a mesma coisa que a forma como você escreveu seu laço for. Pode ser interessante passar a usar o for/in no futuro :)
```suggestion
  for (let index in array) {
```